### DISCUSSION TITLE
msvc or msys 2 which do i choose?

### DISCUSSION THREAD

**Discussion by MattiasN86** [azahar-emu/azahar#345] [General]
msvc or msys 2 which do i choose?  Because i have no idea, running on windows 10 with a nvidia rtx card.

**Comment by OpenSauce04** [comment]
Generally you should try MSVC first and then use MSYS2 if you run into issues

**Comment by kaz-mp6** [comment]
Would you mind giving me a brief explanation of the difference between the two?

**Reply by Algiuxs** [reply]
MSVC is developed by Microsoft and is closed source, but compile times are faster.
While Msys2 is open source, made by a community of people, compile times are slower, but runtime performance is sometimes faster, and also has more optimization flags.
Msys2 also has a package manager and a Unix shell.
But MSVC has an entire ide Wich is VS(Visual studio), it comes with a debugger and a lot more.

**Reply by ThisIsStupidGitHub** [reply]
When updating, how can we tell what version we previously installed, or does it matter?

**Reply by Algiuxs** [reply]
> When updating, how can we tell what version we previously installed, or does it matter?

It doesn't matter. The program is still the same.
